src/courtroom.cpp

```diff
@@ -4446,7 +4446,7 @@ void Courtroom::on_iniswap_dropdown_changed(int p_index)
   }
   ao_app->write_to_file(
       swaplist.join("\n"),
-      ao_app->get_base_path() + "iniswaps.ini");
+      ao_app->get_real_path(VPath("iniswaps.ini")));
```

Comment:
Same here. Basically this gotcha exists anywhere where you want to write a file that may not exist.